frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts

```diff
@@ -56,11 +56,19 @@ const k8sResourceCapabilityToUISchema = (capability: SpecCapability): UiSchema =
 };
 
 const fieldDependencyCapabilityToUISchema = (capability: SpecCapability): UiSchema => {
-  const [, path, value] = capability.match(REGEXP_FIELD_DEPENDENCY_PATH_VALUE) ?? [];
-  if (!!path && !!value) {
-    return { 'ui:dependency': { path: descriptorPathToUISchemaPath(path), value } };
-  }
-  return {};
+  const [, path, controlFieldValue] = capability.match(REGEXP_FIELD_DEPENDENCY_PATH_VALUE) ?? [];
+  const controlFieldPath = descriptorPathToUISchemaPath(path);
+  const controlFieldName = _.last(controlFieldPath);
+  return {
+    ...(path &&
+      controlFieldValue && {
+        'ui:dependency': {
+          controlFieldPath,
+          controlFieldValue,
+          controlFieldName,
+        },
+      }),
```

Comment:
Update property names to be more descriptive and add field name for easier reference when sorting dependent/control fields.